### PR TITLE
Adding CSS Initial Letter property

### PR DESCRIPTION
**Pull request by jensimmons**
Here's a json file for initial letter. #1926

### CODE REVIEW

File: css-initial-letter.json [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+      "url":"https://bugzilla.mozilla.org/show_bug.cgi?id=1223880",
+      "title":"Firefox Implementation Ticket"
+    },
+    {
+      "url":"https://developer.mozilla.org/en-US/docs/Web/CSS/initial-letter",
+      "title":"MDN"
+    }, 
+    {
+      "url":"https://webdesign.tutsplus.com/tutorials/better-css-drop-caps-with-initial-letter--cms-26350",
+      "title":"Blog post on Envato Tuts+, /"Better CSS Drop Caps With initial-letter/""
```
Comment (cvrebert) [inline css-initial-letter.json]: You want backslashes (`\`) instead of forward-slashes here.

File: css-initial-letter.json [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+    }
+  },
+  "notes":"Firefox status: [in-development](https://bugzilla.mozilla.org/show_bug.cgi?id=1040714)",
+  "notes_by_num":{
+    "1":"Safari implementation is incomplete. Does not allow applying web fonts to the initial letter."
+  },
+  "usage_perc_y":0,
+  "usage_perc_a":0,
+  "ucprefix":false,
+  "parent":"parentfeatureid",
```
Comment (cvrebert) [inline css-initial-letter.json]: The value here should be an empty string, since there's no parent feature.

**Comment by cvrebert** [comment]
Also, the JSON file needs to go under the `/features-json/` directory, not at the top-level of the repo.

**Comment by Fyrd** [comment]
@jensimmons Awesome, thanks! Please address the issues @cvrebert mentioned and I'll merge it. Also, any chance you have a test case for this?

**Comment by jensimmons** [comment]
Thank you @cvrebert! And duh, it was in the wrong place... I've been doing all this on the Github website, and not on a local clone, so I'm getting more things wrong. It's good to learn how to do this from the web though, so thanks for your patience.

I do have test cases: http://labs.jensimmons.com/#initialletter

File: features-json/css-initial-letter.json [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+      "43":"n",
+      "44":"n",
+      "45":"n",
+      "46":"n",
+      "47":"n",
+      "48":"n",
+      "49":"n",
+      "50":"n",
+      "51":"n",
+      "52":"n"
```
Comment (jensimmons) [inline features-json/css-initial-letter.json]: Ah, thanks.

**Comment by Fyrd** [comment]
@jensimmons Great, thanks for all the fixes!

**Pull request merged**